reference/pipeline/stages/index.md

```diff
@@ -37,7 +37,10 @@ the default behavior and re-bake your image each time the pipeline runs, select
 
 ### Canary Analysis
 Run [automated canary analysis](/guides/user/canary/) against the deployment
-before fully deploying.
+before fully deploying. Note that this stage _only_ handles analysis; you're
+responsible for provisioning and cleaning up your canary instances elsewhere.
```

Comment:
Does elsewhere mean in another (downstream) stage?

Reply:
Could drop elsewhere and add another sentence with something like: "This is typically done within the same pipeline via stages surrounding the Canary Analysis stage."

Reply:
Yeah, Matt's wording is much better -- updated it to that.